example/frontend/src/index.ts

```diff
@@ -18,7 +18,7 @@ function getWalletObj(x: WalletObjContainer) {
 const blockfrostInst = new Blockfrost(blockfrostUrl, blockfrostProjId);
 
 // Initialise the lucid instance.
-const lucid = await Lucid.new(blockfrostInst, "Testnet");
+const lucid = await Lucid.new(blockfrostInst, "Preview");
```

Comment:
Need to update the lucid dep for this to work

Reply:
Done!